### PR TITLE
Vector VS Raster

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Fixes https://github.com/CartoDB/cartodb/issues/11783.

### CODE REVIEW

**Comment by alonsogarciapablo** [comment]
@donflopez we need to add `vector:false` to [these options](https://github.com/CartoDB/cartodb/blob/5b582c1d6f61788de84a7148757e2f9919a3bc5b/lib/assets/core/javascripts/cartodb3/editor.js#L279-L288) to ensure vector rendering is disabled in BUILDER. I saw you removed the `vector` flag from vizjson (:+1:), but that will cause CartoDB.js to choose between raster and vector both in BUILDER and embed.

**Comment by donflopez** [comment]
[It is](https://github.com/CartoDB/cartodb/blob/vector-vs-raster-1624/lib/assets/core/javascripts/cartodb3/editor.js#L288), isn't it?

**Comment by alonsogarciapablo** [comment]
> It is, isn't it?

alt text

👍 👍 👍

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "4.7.37",
+  "version": "4.7.214",
```
Comment (javitonino) [inline package.json]: This is a big version bump :P

Comment (alonsogarciapablo) [inline package.json]: Yes! We'll fix this before merging! :+1:

**Comment by alonsogarciapablo** [comment]
👏 👏 👏 👏 👏

**Pull request merged**